.travis.yml

```diff
@@ -1,11 +1,20 @@
+dist: bionic
 language: c
 
 # Setting sudo access to false will let Travis CI use containers
 # rather than VMs to run the tests. For more details see:
 # https://docs.travis-ci.com/user/reference/overview/
 sudo: false
 
+before_install:
+  - sudo apt-get install python3-pip ninja-build
+  - pip3 install --upgrade pip
+  - pip3 install --user meson
+  - meson --version
```

Comment:
Not a big fan of this. Rolling Meson releases make the CI kinda unpredictable. Better to install Meson from Ubuntu, also stuff installed via apt should be done with the apt addon, see https://github.com/stephanlachnit/editorconfig-ctest/blob/d32d7582d551442de77cfd5dd8a6e2f8ad146c57/.travis.yml#L4-L10

Reply:
thanks for the example, will update